scrapers/AEBN.yml

```diff
@@ -10,6 +15,53 @@ movieByURL:
       - aebn.com
     scraper: movieScraper
 xPathScrapers:
+  performerScraper:
+    performer:
+      Name: //div[@class='dts-section-page-heading-title']/h1
+      Gender:
+        selector: //li[@class="section-detail-list-item-gender"]/text()
+        postProcess:
+          - map:
+              TS: transgender_female
+      URL: //link[@rel="canonical"]/@href
+      Birthdate:
+        selector: //div[@class='section-detail dts-list-attributes']/ul/li[contains(.,"Birth Date")]
+        postProcess:
+          - replace:
+              - regex: .+:\s(.+)
+                with: $1
+              - regex: "Sept"
+                with: "Sep"
+          - parseDate: Jan 2, 2006
+      Height:
+        selector: //li[@class='section-detail-list-item-height']/text()
+        postProcess:
+          - replace:
+              - regex: .+\((.+)\)
+                with: $1
+      Weight:
+        selector: //li[@class='section-detail-list-item-weight']/text()
+        postProcess:
+          - replace:
+              - regex: .+\((.+)\)
+                with: $1
+      EyeColor: 
+        selector: //li[@class='section-detail-list-item-eye-color']/text()
+      Ethnicity:
+        selector: //li[@class='section-detail-list-item-ethnicity']/text()
+        postProcess:
+          - map:
+              White: Caucasian
```

Comment:
I think in some of the other scrapers we use the opposite? 
For example the babepedia scraper uses the below
```yaml
          - map:
              Asian: asian
              Caucasian: white
              Black: black
              Latin: hispanic
```

Reply:
I'm basing the scraper entries based on what StashDB has defined here https://github.com/stashapp/stash-box/blob/bae0400549c51ec055920ea326ed736ba8f12987/graphql/schema/types/performer.graphql#L31

So that when people submit them to StashDB they don't have to manually enter the value due to it not getting matched.